pages/linux/loadkeys.md

```diff
@@ -0,0 +1,20 @@
+# loadkeys
+
+> Load keyboard translation tables.
+> More information: <https://manned.org/loadkeys>.
+
+- Load the default keymap:
+
+`loadkeys --default`
+
+- Load default keymap when an unusual keymap is loaded and `-` sign cannot be found:
+
+`loadkeys defmap`
+
+- Load a keymap from the specified file for the console:
+
+`loadkeys --console {{/path/to/console}} {{filename}}`
```

Comment:
is this `/dev/tty1`? we have `/dev/sdXN` as a valid placeholder, so:
```suggestion
`loadkeys --console {{/dev/ttyX}} {{path/to/file}}`
```

Reply:
Yeah it can be /dev/ttyX or just /dev/console